Source/FORMDataSource.h

```diff
@@ -65,6 +65,7 @@ typedef UICollectionViewCell * (^FORMFieldConfigureCellForItemAtIndexPath)(FORMF
 - (NSInteger)numberOfFields;
 - (NSArray *)forms;
 - (NSDictionary *)valuesDictionary;
+- (NSDictionary *)removedDynamicValues;
```

Comment:
Yes please, also in the future adding more functionality to removedValues while been backwards compatible should be easier to do without breaking the API.